### DISCUSSION TITLE
import results

### DISCUSSION THREAD

**Discussion by selarem** [idaholab/moose#23040] [Q&A Modules: Solid mechanics] [answered]
Hello moose community,
I would like to model impact of solid1 on solid2. I have meshed with abaqus both and got some problems because, looks like, having two parts in a mesh does not work in moose. So I am trying to make an elastic analysis with a single part mesh composed of two ELSETs (solid1 and solid2). After a first elastic computation, in the second part of this work, I would like to import displacements but considering exclusively the solid2 elset  during all the new analysis. In fact, solid1 models the impactor that is not needed in the second analysis.

thank you for any help.

Regards,
Sab

**Comment by GiudGiud** [comment]
Hello

What do you mean by having two parts in a mesh does not work in MOOSE?
Do you mean that the Abaqus mesh does not load in MOOSE?

Guillaume

**Comment by selarem** [comment, marked as answer]
Hi,
I remembrement receiving a message saying (almost) that a problem is raised
by libmesh because the mesh is composed of two parts. I think I will try
without impactor, just an imposed displacement on the zone of impact, this
will avoid dealing with contact problems which are not the main concern in
our analysis.

Thanks for reply,
Regards

**Reply by GiudGiud** [reply]
@jwpeterson do you know of plans to support multi-parts from Abaqus? Out of curiosity, @selarem has found a workaround

**Reply by jwpeterson** [reply]
There is currently a hard-coded error for multi-PART Abaqus files in the libmesh AbaqusIO reader:
```
          // 0.) Look for the "*Part" section
          if (upper.find("*PART") == static_cast<std::string::size_type>(0))
            {
              libmesh_error_msg_if
                (_already_seen_part,
                 "We currently don't support reading Abaqus files with multiple PART sections");

              _already_seen_part = true;
            }
```
but I think this is mainly due to not having a good test case in the past to develop the reader from, rather than any fundamental reason why they could not be supported.

I don't know of any current plans to add support for this type of file.
